lib/import/BpmnImporter.js

```diff
@@ -108,7 +108,11 @@ BpmnImporter.prototype.add = function(semantic, di, parentElement) {
     // add a virtual element (not being drawn)
     element = this._elementFactory.createRoot(elementData(semantic, di));
 
-    this._canvas.setRootElement(element);
+    if (is(semantic, 'bpmn:SubProcess')) {
+      element.id = element.id + '_plane';
```

Comment:
Is the following case possible?

```xml
<bpmndi:BPMNDiagram id="BPMNDiagram_1">
  <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_1rjrv55">
    <bpmndi:BPMNShape id="Activity_1_di" bpmnElement="Subprocess" isExpanded="false">
      <dc:Bounds x="160" y="77" width="100" height="80" />
    </bpmndi:BPMNShape>
    <bpmndi:BPMNShape id="Activity_2_di" bpmnElement="Task_A">
      <dc:Bounds x="290" y="77" width="100" height="80" />
    </bpmndi:BPMNShape>
  </bpmndi:BPMNPlane>
</bpmndi:BPMNDiagram>
<bpmndi:BPMNDiagram id="BPMNDiagram_2">
  <bpmndi:BPMNPlane id="BPMNPlane_2" bpmnElement="Subprocess">
    <bpmndi:BPMNShape id="Activity_3_di" bpmnElement="Task_B">
      <dc:Bounds x="420" y="77" width="100" height="80" />
    </bpmndi:BPMNShape>
  </bpmndi:BPMNPlane>
</bpmndi:BPMNDiagram>
<bpmndi:BPMNDiagram id="BPMNDiagram_3">
  <bpmndi:BPMNPlane id="BPMNPlane_3" bpmnElement="Subprocess">
    <bpmndi:BPMNShape id="Activity_4_di" bpmnElement="Task_B">
      <dc:Bounds x="420" y="77" width="100" height="80" />
    </bpmndi:BPMNShape>
  </bpmndi:BPMNPlane>
</bpmndi:BPMNDiagram>
```

Using the `ids` utility would make sure that there are no ID clashes.

Reply:
Generally, this is the first time we don't always have the same ID on the shape and the business object. I'm trying to think of potential issues this could cause but haven't found anything so far. 👍🏻

Reply:
Having multiple Planes for the same Subprocess has implications beyond the ID clash, as it is no longer clear which one should be linked to the Subprocess Navigation. I suggest we don't support this configuration for now.

As for `ids` util: I tried `moddle.ids.nextPrefixed('Plane', element)`, but the `ids` utility is only present in the modeler: https://github.com/bpmn-io/bpmn-js/blob/157aec6ed6e2c5bfde10c6176473d5eda6dad1b9/lib/BaseModeler.js#L50

Is there another Utility that I could use? Creating a separate Ids instance just for import seems like bad practice.

Reply:
Okay, got it. We don't have to tackle this right now but we might just have to introduce an IDs util for the viewer in the future. We actually use a separate IDs util in the properties panel for example: https://github.com/bpmn-io/bpmn-properties-panel/blob/main/src/utils/ElementUtil.js#L28

Reply:
Should I create a follow-up issue to move the ID util into the baseViewer?